src/psij/web/js/BuildAccordianModelFromSites.js

```diff
@@ -0,0 +1,96 @@
+/*
+    This builds the DataModel for the Calendar View.
+    It preps the Data for consumption.
+ */
+PS.BuildAccordianModelFromSites = function() {
+
+    var get_states_ = function( months ) {
+
+        var sts = [];
+        var objStates = [];
+
+        for( var month in months ) {
+
+            var days = months[month];
+            for( var day in days ) {
+
+                var res = days[day];
+
+                var sobj = PS.SitesModel.getMainBranch( res.branches );
+
+                if( sobj === undefined) {
+                    //  case where there are no branches or no main branch.
+                    sobj = res;
+                }
+
+                var bc = globalMethods.badnessClass(sobj);
+                sts.push(bc);
+
+                var branches = PS.SitesModel.getBranchesForSmallBubbles( res.branches );
+
+                objStates.push({
+                    mainState: bc,
+                    branches: branches
+                });
+            }
+        }
+
+        return objStates;
+    };
+
+
+    var get_ = function( accordianModelRef ) {
+
+        //  Accordian Model.
+        var am = [];
+        var machObjs = {};
+
+        for( var x=0; x < PS.sites.length; x++ ) {
+
+            var site = PS.sites[x];
+            var domain = globalMethods.getDomain(site.site_id);
+
+            machObjs[domain] = machObjs[domain] || [];
+            machObjs[domain].push({
+                machine: site.site_id,
```

Comment:
I don't think anything has changed:
https://github.com/ExaWorks/psij-testing-service/blob/pa-accordian/src/psij/web/js/BuildAccordionModelFromSites.js#L60

Reply:
ok, now it should be ok.

Reply:
Sorry for not being clear, but `domain` was fine.  `siteId` is clearly not the site id.
'machine' on the other hand, it is not quite clear why we need a new term for what is the site id, so that should be changed.

I didn't look at the snippet above. I checked the actual file in the branch and the change wasn't there.

Reply:
You have:
```
var siteId = globalMethods.getDomain(site.site_id);
```

the variable `siteId` is not the same as `site.site_id`. You call `getDomain` to get the domain from the site id, so whatever is returned should not be put in something called `siteId` because it is NOT the same as the site id.